tests-fuzz/targets/ddl/fuzz_alter_table.rs

```diff
@@ -179,6 +198,28 @@ async fn execute_alter_table(ctx: FuzzContext, input: FuzzInput) -> Result<()> {
         let mut columns = table_ctx.columns.clone();
         columns.sort_by(|a, b| a.name.value.cmp(&b.name.value));
         validator::column::assert_eq(&column_entries, &columns)?;
+
+        // Validates table options
+        let sql = format!("SHOW CREATE TABLE {}", table_ctx.name);
+        let mut table_options = validator::table::fetch_table_options(&ctx.greptime, &sql).await?;
+        table_options.sort_by(|a, b| a.key().cmp(b.key()));
+        let mut expected_table_options = table_ctx.table_options.clone();
+        expected_table_options.sort_by(|a, b| a.key().cmp(b.key()));
+        table_options
+            .iter()
+            .zip(expected_table_options.iter())
+            .for_each(|(a, b)| {
+                if let (
+                    AlterTableOption::TwcsMaxOutputFileSize(a),
+                    AlterTableOption::TwcsMaxOutputFileSize(b),
+                ) = (a, b)
+                {
+                    // File size is not exact, so we compare string representation
+                    assert_eq!(a.to_string(), b.to_string());
```

Comment:
We should compare the `ReadableSize` struct if possible.

Reply:
Currently the file size is extracted and parsed from `SHOW CREATE` results, which is not exact. I wonder if there's some method to get the exact `TwcsMaxOutputFileSize` through sql interface.

Reply:
If the string representations are equal, why does the file size extracted is not exact?

Reply:
Seems a float point issue, example:
The fuzzer set `ALTER TABLE `AUt` SET 'compaction.twcs.max_output_file_size' = '11547.5PiB'`, and read 11547.5PiB as well, but the `u64` inside the readable size is:
![image](https://github.com/user-attachments/assets/0cadb78a-06ae-4a35-aad1-5f2666474b38)
So the assertion failed in a weird way:
![image](https://github.com/user-attachments/assets/b9db2888-2598-4f29-8607-e26b2edac12f)